src/mscorlib/src/System/Threading/SpinLock.cs

```diff
@@ -277,7 +277,7 @@ public void TryEnter(int millisecondsTimeout, ref bool lockTaken)
         /// <summary>
         /// Try acquire the lock with long path, this is usually called after the first path in Enter and
         /// TryEnter failed The reason for short path is to make it inline in the run time which improves the
-        /// performance. This method assumed that the parameter are validated in Enter ir TryENter method
+        /// performance. This method assumed that the parameter are validated in Enter or TryEnter method. ^^ir:or?
         /// </summary>
```

Comment:
Need help with this one: ir .... or?

Reply:
*or*. Ideally `<see>` should be used with `Enter` and `TryEnter` though as I said on a similar case with corefx, maybe just focus on spelling mistakes in this PR and leave markup for another.